charts/woodpecker/charts/agent/values.yaml

```
@@ -49,19 +50,35 @@ extraVolumes: []
   #     claimName: example
 
 # -- Additional volumes that will be attached to the agent container
-extraVolumeMounts: []
+extraVolumeMounts:
+  []
```

This formatting is wrong. Sometimes cause by prettier due to the wrong indentation in the comment block below.

```suggestion
extraVolumeMounts: []
```